templates/authorDashboard/authorDashboard.tpl

```diff
@@ -77,6 +82,9 @@
 			<tab id="publication" label="{translate key="submission.publication"}">
 				<div class="pkpPublication" ref="publication" aria-live="polite">
 					<pkp-header class="pkpPublication__header" :is-one-line="false">
+						<span class="pkpPublication__changeSubmissionLanguage">
```

Comment:
The styling for the new information doesn't match with the existing patterns here.

<img width="1047" alt="Original publications tab" src="https://github.com/user-attachments/assets/32abd60c-d2ae-43c3-b879-9f3ff0be6250">
Original publications tab ☝️

<img width="1076" alt="Publications tab with change locale options" src="https://github.com/user-attachments/assets/d46ce0de-3481-4022-a334-de1f827a7cab">
Publications tab with change locale options ☝️

This goes for both the authorDashboard here and the workflow template. @Devika008, do you have any advice or thoughts on how this information should be presented to have it fit with the rest of the design/styling here?

Reply:
@ewhanson, what do you exactly mean?

Reply:
@bozana, with the `Status: Unscheduled` the "key," `Status` is bold and the "value," Unscheduled is a normal font weight and coloured. With the submission language it's reversed. The "key," "Current Submission Language" is a normal font weight, and the "value," the language, is bold.

Reply:
In Devika's mock-ups in Figma it is so, so we will leave it for now, till she answers...